openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -12,6 +15,28 @@
 logger = logging.getLogger(__name__)
 
 
+class IngestionError(Exception):
```

Comment:
We have a couple of places where custom exceptions are defined:
```bash
$ rg -g "*.py" "^class .*Exception"
openverse_catalog/dags/common/requester.py
23:class RetriesExceeded(Exception):

openverse_catalog/dags/common/licenses/licenses.py
21:class InvalidLicenseURLException(Exception):

openverse_catalog/dags/retired/common/pg_cleaner.py
79:class CleaningException(Exception):

openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py
18:class IngestionError(Exception):
```

Do you think there's any value in centralizing these into a `common/exceptions.py` file?

Reply:
Hm that's a good point. I think each of those custom exceptions are only used within their respective module, with the exception of `RetriesExceeded` being used once here. I don't imagine the `IngestionError` would be used outside of this module, either, although maybe a use case will come up. I think I'm slightly biased to keeping them where they are, at least until that situation changes.

Reply:
That works for me! You're right, each of these are pretty context specific. If it becomes super hairy down the line maybe we can centralize them then.